include/kwk/utility/joker.hpp

```diff
@@ -27,6 +44,18 @@ namespace kwk
     KWK_TRIVIAL friend constexpr joker  operator*(joker, joker) noexcept { return {}; }
     KWK_TRIVIAL friend constexpr joker  operator*(joker,  auto) noexcept { return {}; }
     KWK_TRIVIAL friend constexpr joker  operator*(auto , joker) noexcept { return {}; }
+
+    std::int8_t integral_value_type_size = KWK_DEFAULT_AXIS_BITNESS;
+
+    template <typename Integral>
+    constexpr joker operator[](Integral /*const v*/) const noexcept { /*ct_assert(v == 0)*/ return { std::int8_t(sizeof(Integral)) * (std::is_signed_v<Integral> ? -8 : 8 )}; }
```

Comment:
this is redudant with as<T> beign a proper extent as explained in the doc.

Reply:
>this is redudant with as beign a proper extent as explained in the doc.
as said above - am in the process of changing my stuff of working with ass and reverting this as we speak...